services/dashboard/views/apps/compose/popups/choose_edge.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal" tabindex="-1">
-  <div class="modal-dialog modal-md">
+  <div class="modal-md col-md-5 col-md-offset-3">
```

Comment:
Tried, "text-center" looks not good.
![modal](https://cloud.githubusercontent.com/assets/2595532/12381997/af75d536-bdcf-11e5-91b5-ffc267770aea.jpg)